test/test.api.js

```diff
@@ -649,6 +650,17 @@ DRIVERS.forEach(function(driverName) {
                 storeName: 'storagename2'
             });
 
+            // Same name, but different storeName since this has been malfunctioning before w/ IndexedDB.
+            localforage3 = localforage.createInstance({
+                name: 'storage2',
+                // We need a small value here
+                // otherwise local PhantomJS test
+                // will fail with SECURITY_ERR.
+                // TravisCI seem to work fine though.
+                size: 1024,
+                storeName: 'storagename3'
+            });
+
             Promise.all([
                 localforage.setDriver(driverName),
                 localforage2.setDriver(driverName)
```

Comment:
It lacks the following:

```javascript
localforage3.setDriver(driverName)
```
I think that's why tests pass whereas they should fail, depending on the driver used by default.